A32NX/html_ui/Pages/A32NX_Core/A32NX_TransitionAltitude.js

```diff
@@ -0,0 +1,345 @@
+class A32NX_TransitionAltitude {
+    init() {
+        console.log('A32NX_TransitionAltitude init');
+        this.currentDeparture = "";
+        this.currentArrival = "";
+        this.offline = false;
+        this.checkStart = true;
+        this.checkStartSec = 0;
+        this.testAirport = "";
```

Comment:
**question**: What does `testAirport` mean?

Reply:
It is test airport that our online api is working correctly. Check KLAX TA, if there is no response in 30 sec, using offline db in pr